.detoxrc.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
     emulator: {
       type: 'android.emulator',
       device: {
-        avdName: 'Pixel_6_API_30'
+        avdName: 'Pixel_6_API_30_1'
```

Comment:
Do we need to add an `avdName` to support testing the 32-bit OS Android architecture many of our users require?

I believe that the Xiaomi Redmi product line is widely adopted by zcash/zingo users.

Reply:
uppppsssss, I changed that! Well is not a big deal because it is only the `description` of a emulator, means nothing. I'm not familiar with Xiaomi devices, but I think we have to looking for an emulator for this kind of device, if exists... we can emulate it with 32 bits arch. In local & in CI... I guess.

Reply:
But I don't think this is the goal of this PR...